Comp2025/src/main/java/frc/robot/subsystems/Vision.java

```diff
@@ -104,63 +104,40 @@ public void initialize( )
   {
     DataLogManager.log(String.format("%s: Subsystem initialized!", getSubsystem( )));
 
-    LimelightHelpers.setLEDMode_ForceOff(kLLName);
-    LimelightHelpers.setStreamMode_PiPSecondary(kLLName);
+    LimelightHelpers.setLEDMode_ForceOff(Constants.kLLLeftName);
+    LimelightHelpers.setLEDMode_ForceOff(Constants.kLLRightName);
+    LimelightHelpers.setStreamMode_PiPSecondary(Constants.kLLLeftName);
+    LimelightHelpers.setStreamMode_PiPSecondary(Constants.kLLRightName);
 
-    SetIMUMode(imuMode.EXTERNAL_SEED);
+    SetIMUMode(imuMode.EXTERNAL_SEED, "limelight-left");
   }
 
   /****************************************************************************
    * 
-   * rotateCameraStreamMode - rotate through different stream modes
-   */
-  public void rotateCameraStreamMode( )
-  {
-    switch (m_stream)
-    {
-      default :
-      case PIPSECONDARY :
-        m_stream = streamMode.PIPMAIN;
-        LimelightHelpers.setStreamMode_PiPMain(kLLName);
-        break;
-      case PIPMAIN :
-        m_stream = streamMode.STANDARD;
-        LimelightHelpers.setStreamMode_Standard(kLLName);
-        break;
-      case STANDARD :
-        m_stream = streamMode.PIPSECONDARY;
-        LimelightHelpers.setStreamMode_PiPSecondary(kLLName);
-    }
-
-    DataLogManager.log(String.format("%s: Set stream mode (setStreamMode_PiPxxx) %s", getSubsystem( ), m_stream));
-  }
-
-  /****************************************************************************
-   * 
-   * Limelight auto-aiming control for rotational velocity.
+   * Limelight auto-aiming control for rotational velocity. Only aligns the left Limelight.
    * 
    * @param maxAngularRate
    *          max angular rate to scale against
    * @return desired proportional angular velocity to rotate the chassis
    */
   public AngularVelocity aimProportional(AngularVelocity maxAngularRate)
   {
-    double proportionalFactor = -LimelightHelpers.getTX(kLLName) * kAimingKp;
+    double proportionalFactor = -LimelightHelpers.getTX(Constants.kLLLeftName) * kAimingKp;
```

Comment:
Might as well have these take a limelight name to make them symmetrical with the other methods.